sharding-proxy/src/main/java/io/shardingsphere/shardingproxy/runtime/GlobalRegistry.java

```diff
@@ -214,4 +218,26 @@ public synchronized void renew(final ConfigMapChangedEvent configMapChangedEvent
     public synchronized void renew(final CircuitStateChangedEvent circuitStateChangedEvent) {
         isCircuitBreak = circuitStateChangedEvent.isCircuitBreak();
     }
+    
+    /**
+     * Renew to add new schema.
+     *
+     * @param schemaAddChangedEvent schema add changed event
+     */
+    @Subscribe
+    public synchronized void renew(final SchemaAddChangedEvent schemaAddChangedEvent) {
```

Comment:
When rename SchemaAddChangedEvent, don't forget rename variable and javadoc together

Reply:
More discussions here.